portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,19 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Dangely's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Dangely's Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <p>Click here to get a random fact about me:</p>
+      <button onclick="addRandomFact()">About Me</button>
+      <div id="fact-container"></div>
```

Comment:
Generally, you don't want to use id's unless this div is the only element that will contain this id. For now it should be fine.